### PR TITLE
doc fixes for awful.mouse.resize and awful.rules

### PR DESCRIPTION
**Pull request by blueyed**

### CODE REVIEW

File: lib/awful/rules.lua [hunk 1]
```diff
@@ -457,42 +457,39 @@ function rules.execute(c, props, callbacks)
...
 
     -- Make sure the tag is selected before the main rules are called.
-    -- Otherwise properties like "urgent" or "focus" may fail because they
-    -- will be overiden by various callbacks.
-    -- Previously, this was done in a second client.manage callback, but caused
-    -- a race condition where the order the require() would change the output.
+    -- Otherwise properties like "urgent" or "focus" may fail (if they were
+    -- overridden by other callbacks).
+    -- Previously this was done in a second client.manage callback, but caused
+    -- a race condition where the order of modules being required would change
```
Comment (psychon) [inline lib/awful/rules.lua]: Perhaps change `required` to `loaded` here? The old comment was a bit more precise with `require()`.

File: lib/awful/mouse/resize.lua [hunk 2]
```diff
@@ -89,7 +89,7 @@ function module.add_leave_callback(cb, context)
     table.insert(callbacks.leave[context], cb)
 end
 
--- Resize, the drawable.
+-- Resize the drawable.
```
Comment (Elv13) [inline lib/awful/mouse/resize.lua]: Why?

Comment (Veratil) [inline lib/awful/mouse/resize.lua]: Comma there is grammatically incorrect.

File: lib/awful/rules.lua [hunk 3]
```diff
@@ -457,42 +457,39 @@ function rules.execute(c, props, callbacks)
...
 
-    -- Now that the tags and screen are set, handle the geometry
+    -- Handle the geometry (since tags and screen are set).
     if props.height or props.width or props.x or props.y or props.geometry then
         rules.extra_properties.geometry(c, nil, props)
     end
 
-    -- As most race conditions should now have been avoided, apply the remaining
-    -- properties.
+    -- Apply the remaining properties (since race conditions are handled).
```
Comment (Veratil) [inline lib/awful/rules.lua]: Since known race conditions are hopefully are handled.

File: lib/awful/mouse/resize.lua (hunk 2 above)
Comment (Elv13) [inline lib/awful/mouse/resize.lua]: ah, sorry, I saw `---` -> `--` due to the patch format. It should have been `---` all along.

Comment (Veratil) [inline lib/awful/mouse/resize.lua]: It should be a `---`? Right now it's still just a `--`.

Comment (blueyed) [inline lib/awful/mouse/resize.lua]: Will fix it.

**Pull request merged**